components/classroom/index.html

```diff
@@ -1,50 +1,67 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" type="text/css" href="./styles.css" />
     <title>TPEN Classroom Interface</title>
+    <script src="../projects-list/index.mjs" type="module"></script>
+    <script type="module" src="./fetchData.mjs"></script>
+    <style>
```

Comment:
local styles just to make the page readable without polluting your actual real CSS.